tests/test_server.py

```diff
@@ -3,11 +3,21 @@
 from flask_api import status
 import server
 
+# Status Codes
+HTTP_200_OK = 200
+HTTP_201_CREATED = 201
+HTTP_204_NO_CONTENT = 204
+HTTP_400_BAD_REQUEST = 400
+HTTP_404_NOT_FOUND = 404
+HTTP_409_CONFLICT = 409
+
+
 class TestServer(unittest.TestCase):
     """ Shopcarts Server Tests """
-
     def setUp(self):
         self.app = server.app.test_client()
+        server.Shopcart(1).save()
```

Comment:
I don't think this should be in `setUp` cause not all tests will need this. Also, do you need to get to `Shopcart` via `server` or can you just import it directly?

Reply:
I copied this from the professor's server test code. 
https://github.com/nyu-devops/lab-restful-flask/blob/5f8a987ef5c2e75396a98055ecd0a6fbc23f6055/tests/test_server.py#L27

I suppose because future unit tests on our restful API will require some dummy data